src/bootstrap.ts

```diff
@@ -135,15 +135,16 @@ const setJwtPublicKey = (keyUrl: string) => {
 	getKey(1);
 };
 
-const setupRxNormConnection = (conf: RxNormDbConfig) => {
-	if (!conf.host) return;
+const setupRxNormConnection = (config: RxNormDbConfig) => {
+	if (!config.host) return;
+	const { database, host, password, user, port, connectTimeout } = config;
 	const pool = initPool({
-		database: conf.database,
-		host: conf.host,
-		password: conf.password,
-		user: conf.user,
-		port: conf.port,
-		timeout: conf.timeout,
+		database,
+		host,
+		password,
+		user,
+		port,
+		connectTimeout,
```

Comment:
MySql2 has Types so there are a few updates to align our code with their library